odoo/addons/base/tests/test_gettext_alias.py

```diff
@@ -0,0 +1,48 @@
+import inspect
+from unittest.mock import patch
+
+from odoo.tests import common
+from odoo.tools.translate import GettextAlias
+
+
+class TestGettextAlias(common.TransactionCase):
+    @classmethod
+    def setUp(self):
+        super().setUpClass()
```

Comment:
I don't think this is needed, since your inherited class already does this by default.